modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ActivityQuestionRow.es.js

```diff
@@ -57,6 +58,19 @@ export default function ActivityQuestionRow({
 	const isRowSelected = question.friendlyUrlPath === rowSelected;
 
 	const messageType = useMemo(() => {
+		if (question.showAsAnswer === true) {
+			return {
+				isBestAnswer: true,
+				label: Liferay.Language.get('best answer'),
+				symbol: 'check-circle-full',
+				text: stripHTML(question.articleBody).replace(
+					MESSAGE_TYPES.answer.prefix,
+					''
+				),
+				type: MESSAGE_TYPES.reply.type,
+			};
+		}
```

Comment:
I think we should follow the pattern and create another MESSAGE_TYPE for bestAnswer.

My suggestion is to create: 

```
bestAnswer: {type: 4},
```

```suggestion
		if (question.showAsAnswer) {
			return {
				label: Liferay.Language.get('best-answer'),
				symbol: 'check-circle-full',
				text: stripHTML(question.articleBody).replace(
					MESSAGE_TYPES.answer.prefix,
					''
				),
				type: MESSAGE_TYPES.bestAnswer.type,
			};
		}
```

You need to create a language property for best-answer